src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -1107,4 +1110,13 @@ protected T computeNext() {
                 .filter(name -> !unique.add(name))
                 .collect(Collectors.toSet());
     }
+
+    /**
+     * Return the given {@code dateTime} formatted as string for display.
+     * @param dateTime the datte/time to be formatted
+     * @return String representing the {@code dateTime}.
+     */
+    public static String getDateTimeForDisplay(final ZonedDateTime dateTime) {
+        return dateTime.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.LONG));
```

Comment:
Since `DateTimeFormatter` is immutable and thread safe, you can store the result of `DateTimeFormatter.ofLocalizedDateTime()` in a static final field and avoid the creation every time this method is called.